app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -17,7 +17,7 @@
 %>
 <div class="govuk-document-footer<%= direction_class %>">
   <h2 class="visuallyhidden">Document information</h2>
-  <div class="history-information">
+  <div class="history-information" id="history">
```

Comment:
The "see all updates" link in the metadata points to this anchor.

![image](https://cloud.githubusercontent.com/assets/6908677/14671059/23ee34d0-06f0-11e6-8729-eca65208e2db.png)